### PR TITLE
Added color wheel that have QMK HSV values.

### PR DESCRIPTION
**Pull request by elijahblake81**
I've converted the 360 color wheel to the 255 color wheel for QMK HSV values.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
I think it's very good. But it's a little big. For example, you can adjust the size by doing the following:

```html
<img src="URL_TO_IMAGE_FILE" alt="ALT_TEXT" width="50%"/>
```

File: docs/feature_rgblight.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ Changing the **Hue** cycles around the circle.<br>
 Changing the **Saturation** moves between the inner and outer sections of the wheel, affecting the intensity of the color.<br>
 Changing the **Value** sets the overall brightness.<br>
 
+![QMK Color Wheel with HSV Values](https://i.imgur.com/vkYVo66.jpg)
```
Comment (spidey3) [inline docs/feature_rgblight.md:44]:
```suggestion
Note that while usually the HSV color space uses 0 – 360° for hue, in QMK the scale is from 0 – 255. You can convert by calculating `255 * Hue / 360`. Here is a color wheel in the 0 – 255 hue range for reference:
![QMK Color Wheel with HSV Values](https://i.imgur.com/vkYVo66.jpg)
```

Comment (mtei) [reply docs/feature_rgblight.md:44]: The new picture illustrates only about Hue. So, I think it's worth keeping the previous picture.

Comment (elijahblake81) [reply docs/feature_rgblight.md:44]: why not have both.  1 has the values you can assign to actually use.  The other just explains how the colors are defined but doesn't have any number to associate to them.

Comment (Erovia) [reply docs/feature_rgblight.md:44]: Would it be possible to have this as an svg or at least a png with transparent background?

Comment (tzarc) [reply docs/feature_rgblight.md:44]: Given the usefulness of this in general, should we get this in in its current form, and fix it up later?

Comment (tzarc) [reply docs/feature_rgblight.md:44]: Given the usefulness of this in general, should we get this in in its current form, and fix it up later?

Comment (drashna) [reply docs/feature_rgblight.md:44]: I think so, yeah.

**Review by tzarc — APPROVED** [review]

**Pull request merged**